server.js

```diff
@@ -0,0 +1,4 @@
+const app = require('./lib/rickAndMortyNotesApp');
+const http = require('http');
+
+http.createServer(app).listen(7890);
```

Comment:
```suggestion
http.createServer(app).listen(7890, () => console.log('Listening on 7890'));
```
I find it helpful to have a console log to confirm that my server is up and running.